tests/bluetooth/host/buf/bt_buf_get_rx/src/main.c

```diff
@@ -5,9 +5,9 @@
  */
 
 #include <zephyr/zephyr.h>
-#include <zephyr/bluetooth/hci.h>
 #include <zephyr/bluetooth/buf.h>
 #include "kconfig.h"
+#include "net_buf.h"
```

Comment:
What criteria do you use to select between `< >` and `" "` includes?

Reply:
Just to refer that the file not part of a library as if it is only local to the project.
Technically, the compiler will handle it as if it is `<>` as the header file doesn't reside in the same directory